newIDE/app/src/AssetStore/PrivateAssets/PrivateAssetPackInformationPage.js

```diff
@@ -222,7 +280,7 @@ const PrivateAssetPackInformationPage = ({
         }
       })();
     },
-    [id, sellerId, privateAssetPackListingData.appStoreProductId]
```

Comment:
Do you know why this dependency was here?

Reply:
I have no idea! And I also don't understand why it wasn't complaining